nailgun/entities.py

```diff
@@ -6686,6 +6686,16 @@ def __init__(self, server_config=None, **kwargs):
         }
         super().__init__(server_config, **kwargs)
 
+    def read(self, entity=None, attrs=None, ignore=None, params=None):
+        """Override :meth:`nailgun.entity_mixins.EntityReadMixin.read` to ignore
+        the ``created_at and updated_at``
+        """
+        if ignore is None:
```

Comment:
Non-blocking, but a pythonic suggestion. `ignore = ignore or set()` will handle the None condition, and if its already an empty set/list/dict, it just creates a new set.